scripts/gateway-setup-windows.ps1

```
@@ -0,0 +1,183 @@
+
+    # 检查是否运行
+    try {
+        $process = Get-CimInstance Win32_Process | Where-Object { $_.ExecutablePath -eq $exePath } | Select-Object -First 1
+        if (-not $process) {
+            $process = Get-Process -Name "quantclaw" -ErrorAction SilentlyContinue | Where-Object { $_.Path -and $_.Path -eq $exePath } | Select-Object -First 1
+        }
+    } catch {
+        $process = Get-Process -Name "quantclaw" -ErrorAction SilentlyContinue | Select-Object -First 1
+    }
+    if ($process) {
+        Write-Host "✓ Gateway 进程正在运行 (PID: $($process.Id))" -ForegroundColor Green
```

PSScriptAnalyzer (1.25.0)

[warning] File 'gateway-setup-windows.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)